mlir-tensorrt/compiler/lib/Compiler/StableHloToExecutable.cpp

```diff
@@ -299,16 +251,18 @@ void StableHloToExecutableTask::buildPostClusteringPipeline(
   populateExtensionPasses(pm, opts, Phase::ExecutorLowering);
 
   ConvertCUDAToExecutorPassOptions cudaToExecutorOpts;
-  cudaToExecutorOpts.indexBitwidth = opts.executorIndexBitwidth;
-  cudaToExecutorOpts.usePackedMemRefCConv = opts.executorUsePackedMemRefCConv;
+  cudaToExecutorOpts.indexBitwidth =
+      opts.get<ExecutorOptions>().executorIndexBitwidth;
```

Comment:
Nit: we can probably update the names to leave off the `executor` prefix since it is now obvious

Reply:
Ah yes, I did that with device options but forgot to do so for the executor. Will fix.